src/scheduler.c

```diff
@@ -952,7 +955,9 @@ void scheduler_write_cell_dependencies(struct scheduler *s, int verbose,
     /* and their dependencies */
     for (int j = 0; j < ta->nr_unlock_tasks; j++) {
       const struct task *tb = ta->unlock_tasks[j];
-
+      if(tb->subtype == task_subtype_gpu_unpack
+        		  || tb->subtype == task_subtype_gpu_unpack_f
+      			  || tb->subtype == task_subtype_gpu_unpack_g)continue;
```

Comment:
Why are we skipping this?

Reply:
It was a temp fix to stop the code from crashing because unpack tasks do not have cells. I was trying to see cell level dependencies to debug the code. I will remove this